bleach/_vendor/pip_install_vendor.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 
 
+# check that we're using the pip version used to vendor the file
+test -eq "$(pip --version | cut -d ' ' -f 2)" "20.2.3"
```

Comment:
Tell me more about this line. Why did this get added?

Reply:
There's a single line difference in the dist info returned depending on the pip versions. I'll change it to a comment.

Reply:
Added comment to readme instead